04_numeronym/mohitnag/main_test.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNumeronyms(t *testing.T) {
+	testData := []struct {
+		Scenario string
+		input    []string
+		expected []string
+	}{
+		{"Scenario One", []string{"ehi!@  45%!"}, []string{"e9!"}},
```

Comment:
I think I have mentioned this on an earlier pull request of yours as well, but it is best to use descriptive explanations for your test scenario. Scenario One and Scenario Two are not descriptive and won't give much to anyone else who runs your tests and sees this on the output.

Please reword these so they are explaining what the test is doing.

Eg. Empty String. String with Length greater than 3.

On top of this I would like to see more test cases. Think about the edge cases here, what could possibly go wrong. The function is doing some different input on a string of length 3 vs a string of length >3. Perhaps you want to test around this line of len(3)?

Reply:
Done